Doc/library/time.rst

```diff
@@ -572,12 +579,27 @@ The module defines the following functions and data items:
 
 .. function:: time()
 
-   Return the time in seconds since the epoch as a floating point number.
+   Return the time in seconds since the epoch as a floating point
+   number. The specific date of the epoch and the handling of
+   `leap seconds <https://en.wikipedia.org/wiki/Leap_second>`_
+   is platform dependent.
+   On Windows and most Unix systems, the epoch is January 1, 1970,
+   00:00:00 (UTC) and leap seconds are not counted towards the time
+   in seconds since the epoch. This is commonly referred to as
+   `Unix time <https://en.wikipedia.org/wiki/Unix_time>`_.
```

Comment:
IMHO it's worth it to repeat here:

To find out what the epoch is on a given platform, look at ``gmtime(0)``.